yaml-tests/src/test/resources/aggregate-empty-table.yamsql

```diff
@@ -36,7 +36,9 @@ test_block:
     -
       - query: select count(*) from T1;
       - explain: "SCAN(<,>) | TFILTER T1 | MAP (_ AS _0) | AGG (count_star(*) AS _0) | ON EMPTY NULL | MAP (coalesce_long(_._0._0, promote(0l AS LONG)) AS _0)"
+      - maxRows: 1
```

Comment:
Should similar tests be added for the other aggregate types (sum, max, etc)?

Reply:
I _converted_ a test doing `sum(...)` in the same yaml. This test ensures that `NULL_ON_EMPTY` also works without coalescing to a `0`.